### PR TITLE
Luke changes

### PR DESCRIPTION
**Pull request by maggyb**
@lukefrake please can you review this for me? I've made some style changes I'd like you to check - thanks

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -4,16 +4,16 @@
...
 
 	<ul class="article center columns medium-10 large-8">
 		{% for people in site.data.team %}
 			{% assign person = people[1][0] %}
 			{% if person.name and person.title and person.blurb and person.photo %}
 				<li class="row article_row">
 					<div class="columns small-12 medium-6">
-						<img class="article_image" src="https://www.gravatar.com/avatar/{{ person.photo }}?s=100" alt="Image of {{ person.name }}" title="Headshot of {{ person.name }}">
-						<div class="article_information">
+						<img class="article_image " src="https://www.gravatar.com/avatar/{{ person.photo }}?s=100" alt="Image of {{ person.name }}" title="Headshot of {{ person.name }}">
```
Comment (f000) [inline team.html]: redundant space in class=""

File: team.html [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
+						<div class="article_information ">
```
Comment (f000) [inline team.html:16]: redundant space in class=""

Comment (maggyb) [inline team.html:16]: Thanks!

**Comment by lukefrake** [comment]
👍 Looks great!

**Pull request merged**